package.json

```diff
@@ -31,19 +43,20 @@
     "url": "https://github.com/thgh/rollup-plugin-livereload"
   },
   "files": [
-    "dist",
-    "index.d.ts"
+    "dist"
   ],
   "engines": {
-    "node": ">=8.3"
+    "node": ">=16"
```

Comment:
Fields like `type` and `exports` (needed for ESM) are not supported by older versions of Node.js, so I've bumped the version to the latest supported LTS.